install_ant-media-server.sh

```diff
@@ -85,7 +88,7 @@ fi
 $SUDO apt-get update -y
 check $?
 
-$SUDO apt-get install openjdk-8-jdk unzip jsvc -y
+$SUDO apt-get install openjdk-8-jdk unzip jsvc iptables-persistent -y
```

Comment:
We install iptables-persistent after we use them at line 14 above. This seems cause a problem?